airqo-platform/AirQo-frontend
# [Analytics] Added settings page

#### Summary of Changes (What does this PR do?)
- Created a folder under pages and added files; profiledetails.jsx, and components; circularprogressindicator.jsx and togglebutton.jsx
- Under the pofiledetails.jsx page, there are two tabs that display content namely; profile details and email settings tab
- There are a series of text inputs, buttons and divider lines on both tab displays.
- The component circularprogressindicator.jsx works for only the 3 toggle buttons on the email settings tab and it contains a funstion that counts the number of buttons toggled and displays that number while showing radial progress in a blue color.

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?

- Verify that the components render without error by running "npm run dev".
- Interact with the text input fields by typing "Sample Text" and observe correct behavior.
- Click the buttons or hover over them to ensure that the cursor turns to pointer, among other expected actions.
- Check the styling, layout, and responsiveness of the components by adjusting the window size

#### What are the relevant tickets?

- #1375
- [AN-517](https://airqoteam.atlassian.net/browse/AN-517?atlOrigin=eyJpIjoiNmI3MTRjNGYwMDBmNDc3M2IxZmZhNDg2MzEyOWExOGYiLCJwIjoiaiJ9)

#### Screenshots (optional)
![Screenshot from 2023-08-14 14-46-21](https://github.com/airqo-platform/AirQo-frontend/assets/110837740/9b3a6b6e-92d1-4804-9420-fc94b4f6a798)
![Screenshot from 2023-08-14 14-46-38](https://github.com/airqo-platform/AirQo-frontend/assets/110837740/d48538e6-be16-485c-930e-3433f2faba1a)
![Screenshot from 2023-08-14 14-46-07](https://github.com/airqo-platform/AirQo-frontend/assets/110837740/a552d983-1523-48ad-92ad-78a30840657b)
![Screenshot from 2023-08-14 14-46-28](https://github.com/airqo-platform/AirQo-frontend/assets/110837740/d65fb4d2-d649-48b6-8ff9-519bfde999b3)


[AN-517]: https://airqoteam.atlassian.net/browse/AN-517?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
